Crucible.GameAPI/CrucibleCustomerNpcTemplate.cs

```diff
@@ -120,14 +120,14 @@ public static CrucibleCustomerNpcTemplate CreateCustomerNpcTemplate(string name,
 
             var template = ScriptableObject.CreateInstance<NpcTemplate>();
 
-            template.spawnChance = 1f;
+            //template.spawnChance = 1f;
```

Comment:
I was unable to find an existing counterpart to this field. Commented out for now.